schemas/JSON/manifests/v1.2.0/manifest.defaultLocale.1.2.0.json

```diff
@@ -35,6 +35,21 @@
           "description": "The agreement URL."
         }
       }
+    },
+    "Documentation": {
+      "type": "object",
+      "properties": {
+        "DocumentDescription": {
+          "type": [ "string", "null" ],
+          "minLength": 1,
+          "maxLength": 100,

```

Comment:
The intent here is more of a concise description of what the document provides. Examples that I can think of may be -
* Troubleshooting guide
* Frequently asked questions
* User manual
* Instructions on how to install and use ABC or XYZ plugin with PackageName
* System Requirements

I think 100 would be more than enough, but perhaps this could be made 256 like a short description. I don't think we need an entire document here

Reply:
I guess it's more like a title rather than the contents then, we use Label in the manifests for this concept. maybe DocumentLabel?

Reply:
DocumentLabel
DocumentTitle
DocumentName
DocumentType

I think any of them would work; I know that denelon had suggested DocumentDescription in the original issue, but I'm sure it's open to interpretation

Reply:
Thanks for the suggestions.  Decided to go with DocumentLabel as this more closely aligns with the existing pattern (i.e. AgreementLabel)